pkgs/applications/office/libreoffice/src-still/override.nix

```diff
@@ -6,8 +6,8 @@ attrs:
   '';
   configureFlags = attrs.configureFlags ++ [
     (lib.enableFeature kdeIntegration "kf5")
-    "--with-commons-logging-jar=${commonsLogging}/share/java/commons-logging-1.2.jar"
-    "--without-system-qrcodegen"
+    "--without-system-zxing"
```

Comment:
Why are we not using the system libraries?

Reply:
Because the system library is missing a header file and libreoffice won't compile.
It ships its own version of zxing (which is found in the long download.nix file)

Reply:
It seems that the system zxing is not missing anything and just has its .pc file screwed up, I’ll have a look.

Reply:
Ouch, this is worse than I thought. KhronosGroup/SPIRV-Tools#3905 is a similar bug from October 2020 that concerns broken pkg-config files when CMAKE_INSTALL_{INCLUDE,LIB}DIR is absolute (and with the Nixpkgs CMake tooling it is), it’s still unfixed, and the .pc files for spirv-tools in Nixpkgs are similarly broken. Honestly it seems like .pc files in all CMake-built projects should be suspect.

Reply:
ah. interesting. Thanks for clearing that up. When building with the nixos zxing it just complained about a barcode header file not found. 
Libreoffice compiles with its own version correctly though :+1:

Reply:
@gador see #172150 for my attempt at killing the outdated custom zxing (but my machine is likely too puny to build libreoffice so I haven’t tried that yet)

Reply:
Instead of using some version that they vendored which could be missing patches, updates and nixos workaround we should use our version again especially if the pc file bug is fixed.